docker/production.yml

```diff
@@ -139,11 +139,11 @@ services:
     restart: unless-stopped
     depends_on:
       - telescope
-    # This makes nginx swap only the 2 specified environment variables in our template (see:
+    # This makes nginx swap only the specified environment variables in our template (see:
     # https://github.com/docker-library/docs/issues/496) and then reload this configuration
     # (and the associated certificates) every six hours in the background and launches nginx
     # in the foreground.
-    command: /bin/sh -c "envsubst '$$TELESCOPE_HOST $$API_HOST' < /etc/nginx/nginx.conf.template > /etc/nginx/nginx.conf && while :; do sleep 6h & wait $${!}; nginx -s reload; done & nginx -g 'daemon off;'"
+    command: /bin/sh -c "envsubst '$$TELESCOPE_HOST $$API_HOST $$PORTAINER_HOST' < /etc/nginx/nginx.conf.template > /etc/nginx/nginx.conf && while :; do sleep 6h & wait $${!}; nginx -s reload; done & nginx -g 'daemon off;'"
```

Comment:
I don't quite understand this part.
Are we loading `env` from `/etc/nginx/nginx.conf.template` into `/etc/nginx/nginx.conf ` every 6h?

Reply:
For some reason, we're restarting nginx every 6 hours (maybe for a memory leak, I can't remember. @manekenpix might remember?), and when we do, we are taking what's in the env and using those variables to replace things in the template files in order to create the conf files again.

This seems unnecessary?  But I can't remember why we did it.

Reply:
If I remember correctly, we restart `nginx` every 6 hours in case our `ssl` certificates get renewed. This way, we can reload them and avoid a [Microsoft Exchange](https://www.bleepingcomputer.com/news/microsoft/microsoft-exchange-admin-portal-blocked-by-expired-ssl-certificate/) situation.

Reply:
I should add a comment to this effect, I totally forgot.